patches/server/0739-Implemented-OP-level-API.patch

```diff
@@ -0,0 +1,256 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: MCMDEV <john-m.1@gmx.de>
+Date: Thu, 5 Aug 2021 17:00:10 +0200
+Subject: [PATCH] Implemented OP level API
+
+
+diff --git a/src/main/java/net/minecraft/server/dedicated/DedicatedPlayerList.java b/src/main/java/net/minecraft/server/dedicated/DedicatedPlayerList.java
+index 918f5221e94cbc867349c69c83563e225d2fef1d..734e5305d2a059738df20ea9a72dad6ca6c5b810 100644
+--- a/src/main/java/net/minecraft/server/dedicated/DedicatedPlayerList.java
++++ b/src/main/java/net/minecraft/server/dedicated/DedicatedPlayerList.java
+@@ -52,6 +52,17 @@ public class DedicatedPlayerList extends PlayerList {
+         this.saveOps();
+     }
+ 
++    // Paper start - OP level API
++    public int getOperatorLevel(GameProfile profile)    {
++        return this.getServer().getProfilePermissions(profile);
++    }
++
++    public void setOperatorLevel(GameProfile profile, int value)    {
```

Comment:
As the `DedicatedPlayerList#op` method was simply an overwrite, you could now also scratch this method and change the existing `DedicatedPlayerList#op` method to overwrite the "new" op method in `PlayerList` (`PlayerList#op(GameProfile,int)`).